tests/headers/forward_declared_complex_types.hpp

```diff
@@ -0,0 +1,15 @@
+struct Foo;
+
+struct Bar {
+    Foo *f;
+};
+
```

Comment:
Could you also add an empty struct (`struct Baz {}`), and verify that it doesn't produce an opaque enum?